networkd/network_server.ml

```diff
@@ -949,6 +949,12 @@ module Bridge = struct
 			update_config name {(get_config name) with persistent_b = value}
 		) ()
 
+	let set_ovs_igmp_snooping _ dbg ~name ~value =
```

Comment:
We don't need this function, because we can just call `create` even if the bridge already exists. All functions in the interface, including `make_config` and `create` are idempotent, which means that you can safely call them multiple times. The job of `Bridge.make_config` is to update the current configuration of the bridge based on the new configuration that is given, and only do something when (part of) the configuration has changed.